packages/components/Menu/src/MenuTrigger/useMenuTrigger.ts

```diff
@@ -73,6 +75,24 @@ export const useMenuTrigger = (childProps: MenuTriggerChildProps): MenuTriggerSt
     [childOnAccessibilityAction, setOpen],
   );
 
+  const accessibilityPositionInSet = React.useMemo(() => {
+    const baseAccessibilityPositionInSet = context.accessibilityPositionInSet;
```

Comment:
Recommend moving this line after the `if (childAccessibilityPositionInSet) {` statement. No need to initialize this variable here if it's not going to be used.

Reply:
Thanks, I updated it to return the value from the context instead of setting a value and returning it. Same for the other comment.